index.html

```diff
@@ -10,7 +10,9 @@
     <meta name="author" content="">
 
     <title>cardinal designs</title>
-
+    <META NAME="description" CONTENT="Cardinal Designs">
+    <META NAME="keyword" CONTENT="Graphic Design, South Carolina, CHarleston, Park Circle, Advertising. Web Design, Branding, Logos">
```

Comment:
Try not to use uppercase elements. It's better style to use lower case:

`<meta name="keyword" content="....">`